src/process/slices/userSlice.js

```diff
@@ -0,0 +1,50 @@
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+
+import Api from '../api';
+
+export const logIn = createAsyncThunk('user/logIn', async payload => {
+  try {
+    const res = await Api.post(`/auth/sign_in`, payload);
+    return { ...res.data.data, headers: res.headers };
+  } catch (err) {
+    return err.message;
+  }
+});
+
+export const logOut = createAsyncThunk('user/logOut', async payload => {
```

Comment:
@iamdavidpetric anything else to be done in here?

Reply:
done